src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.Windows.cs

```diff
@@ -714,5 +715,28 @@ internal bool IsReplacementCulture
                 return false;
             }
         }
+
+        internal static CultureData GetCurrentRegionData()
+        {
+            Span<char> geoIso2Letters = stackalloc char[10];
+
+            int geoId = Interop.Kernel32.GetUserGeoID(Interop.Kernel32.GEOCLASS_NATION);
+            if (geoId != Interop.Kernel32.GEOID_NOT_AVAILABLE)
+            {
+                int geoIsoIdLength = Interop.Kernel32.GetGeoInfo(geoId, Interop.Kernel32.GEO_ISO2, ref MemoryMarshal.GetReference(geoIso2Letters), geoIso2Letters.Length, Interop.Kernel32.EN_LANG_ID);
```

Comment:
for the last parameter docs say "Note   The application must set this parameter to 0 if GeoType has any value other than GEO_RFC1766 or GEO_LCID."
https://docs.microsoft.com/en-us/windows/win32/api/winnls/nf-winnls-getgeoinfow

Reply:
we are using GEO_ISO2 and not GEO_RFC1766  nor GEO_LCID.

Reply:
Right, so LangId should be 0, but you're passing Interop.Kernel32.EN_LANG_ID ?

Reply:
The doc kind confused me especially the second part of the following:

```
LangId

Identifier for the language, used with the value of Location. The application can set this parameter to 0, with GEO_RFC1766 or GEO_LCID specified for GeoType. This setting causes the function to retrieve the language identifier by calling GetUserDefaultLangID.
```

I'll change it to 0.